tox.ini

```diff
@@ -27,15 +27,14 @@ deps = -rtests_utils/test-requirements.txt
 commands =
     python setup.py nosetests {posargs}
```

Comment:
This toxenv looks useless to me since code_check and test envs already do the same things.